### PR TITLE
When a juror is deferred or excuse from a paper response the green enter summons reply button still shows

### PR DESCRIPTION
**Pull request by ricardofreitasrocha**
### Change description ###
the processing status is summoned these need to be moved to completed for all paper responses that have moved out of summoned status.

Also occurs when a juror response is deleted (apart of the one year juror_digital housekeeping).

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: server/routes/juror-management/juror-record/juror-record.controller.js [hunk 1]
```diff
@@ -1304,22 +1296,16 @@
...
-
-    if (commonDetails.owner !== '400' && !isCourtUser(req)) {
-      canSummon = false;
-    };
 
     switch (jurorStatus) {
     case 'Undeliverable':
     case 'Responded':
     case 'Completed':
     case 'Disqualified':
```
Comment (akikrahman1) [inline server/routes/juror-management/juror-record/juror-record.controller.js:1304]: do we need to add excused to the list?

File: client/templates/juror-management/_partials/heading.njk [hunk 2]
```diff
@@ -31,7 +31,7 @@
         }) }}
       {% endif %}
 
-      {% if not hasSummons and canSummon %}
+      {% if canSummon %}
```
Comment (akikrahman1) [inline client/templates/juror-management/_partials/heading.njk:34]: The name of this variable might need to be canEnterSummons - just to make it distinguish from can summon jurors to a pool

Comment (ricardofreitasrocha) [reply client/templates/juror-management/_partials/heading.njk:34]: Good point on that :-)

File: server/routes/juror-management/juror-record/juror-record.controller.js (hunk 1 above)
Comment (ricardofreitasrocha) [reply server/routes/juror-management/juror-record/juror-record.controller.js:1304]: That would need confirmation, if we defer someone without entering a response... would then, we never being able to enter a response correct?

File: client/templates/juror-management/_partials/heading.njk (hunk 2 above)
Comment (ricardofreitasrocha) [reply client/templates/juror-management/_partials/heading.njk]: updated

File: server/routes/juror-management/juror-record/juror-record.controller.js (hunk 1 above)
Comment (ricardofreitasrocha) [reply server/routes/juror-management/juror-record/juror-record.controller.js:1306]: updated

**Review by akikrahman1 — APPROVED** [review]

**Pull request merged**